gulp-tasks/wfRegEx.js

```diff
@@ -8,6 +8,7 @@
 var RE_BOOK_PATH = /^book_path: (.*)\n/m;
 var RE_PROJECT_PATH = /^project_path: (.*)\n/m;
 var RE_DESCRIPTION = /^description:\s?(.*)\n/m;
+var RE_BLINK_COMPONENTS = /^  - (Blink>[\w>]+)/gm;
```

Comment:
there are some components that don't start with `Blink>`. Most that we use will however. Instead of regexs, is there a way to get the yaml list and then compare that to the `blinkComponents` array?

```
const usesAnUnknownComponent = blinkComponentsInYaml.filter(c => !blinkComponents.includes(c)).length > 0;
```